documentation/source/users/rmg/input.rst

```diff
@@ -481,6 +481,35 @@ For example ::
 		toleranceMoveSurfaceReactionToCore=5.0,
 	)
 
+Advanced Setting:  Dynamics Criterion Weights
+---------------------------------------------
+In reaction networks reactions that change the number of radicals tend to be much more important
+than other reactions.  Also in many cases specific species within a model are of interest.  
+
+In these sorts of situations RMG implements two different types of weights:
+Reaction Weights: multiplied by the dynamics number for a given reaction at the end of calculations
+Species Weights: multiplied by the contribution of a given species to every dynamics number
+
+The only reaction weights supported are based on the change in the number of radicals, which, can 
+be adjusted by assigning radicalChangeWeight where the weight is calculated W_rxn = exp(radicalChangeWeight*dr) 
```

Comment:
Use double back quotes to emphasize `radicalChangeWeight`

Also use math formating for the equation, and place on a separate line.